website/src/pages/flu/index.astro

```diff
@@ -0,0 +1,7 @@
+---
+import { ServerSide } from '../../views/serverSideRouting';
+
+return Astro.redirect(
+    ServerSide.routing.getOrganismView('flu.sequencing-efforts').pageStateHandler.getDefaultPageUrl(),
```

Comment:
This is wrong ;) Let's use the constants instead.
```suggestion
    ServerSide.routing.getOrganismView(`${Organisms.flu}.${sequencingEffortsViewKey}`).pageStateHandler.getDefaultPageUrl(),
```

Reply:
weird it worked locally but I can change :-)

Reply:
again I used the same configuration as in the other index.astro files, e.g. mpox - is this wrong there?

Reply:
No, the others are correct I think. `sequencing-efforts` is wrong specifically.

Reply:
ah yes I fixed it is now `sequencingEffortsView` :-)